src/commands/dev/dev.js

```diff
@@ -178,18 +178,25 @@ const runCommand = (command, env = {}, spinner = null) => {
   return commandProcess
 }
 
+/**
+ * @typedef StartReturnObject
+ * @property {4 | 6 | undefined=} ipVersion The version the open port was found on
```

Comment:
I'm assuming this is a typo?

```suggestion
 * @property {4 | 6 | undefined} ipVersion The version the open port was found on
```

Reply:
No the equal sign marks the field as optional